client-mvp-04/src/reducers/authReducer.js

```diff
@@ -0,0 +1,18 @@
+import { LOGIN } from '../actions/types';
+
+const authDefaultState = {
+    loggedIn : false
+}
+
+
+export default (state = authDefaultState, {type, payload}) => {
+    switch (type) {
+        case LOGIN:
+            return {
+                ...state,
+                loggedIn : true
+            }
+        default:
+            return state;
+    }
+}
```

Comment:
please add an empty line at the end of the file
https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file